types-fixup.mjs

```diff
@@ -469,6 +478,12 @@ const standardMaterialProps = [
 path = './types/scene/materials/standard-material.d.ts';
 dts = fs.readFileSync(path, 'utf8');
 dts = dts.replace('reset(): void;', 'reset(): void;\n' + getDeclarations(standardMaterialProps));
+dts += `
```

Comment:
I would not mind in code comments for changes in this file as they are not obvious.

Reply:
Added some comments :smile: